tests/beacon/helpers.py

```diff
@@ -36,3 +50,17 @@ def get_pseudo_chain(length, genesis_block):
             parent_root=block.root
         )
         yield block
+
+
+def sign_proof_of_possession(deposit_input, privkey, domain):
+    return bls.sign(deposit_input.root, privkey, domain)
```

Comment:
This is a bit out of the context of this PR, but this probably should take `fork_data` and calculate the `domain` locally using `DOMAIN_DEPOSIT` along with the `fork_data` param. This is instead of forcing the user of this function to pull in `DOMAIN_DEPOSIT` and calculate the domain before passing it.